src/app/app.component.html

```diff
@@ -9,6 +9,20 @@ <h3>Regular textarea
 
 <hr/>
 
+<div>
+    <h3>Dynamic directive - enabled</h3>
+    <textarea [ngModel]="longText"
+              (resized)="onResized($event)"
+              [autosize]="true"></textarea>
```

Comment:
since its an example of dynamic nature of the directive lets have some dropdown or checkbox that controls the state? what do you think?

Reply:
Sure I can add it :)